packages/xrpc-server/src/rate-limiter.ts

```diff
@@ -155,5 +154,5 @@ export const getTightestLimit = (
   return lowest
 }
 
-const defaultKey: CalcKeyFn = (ctx: XRPCReqContext) => getReqIp(ctx.req)
+const defaultKey: CalcKeyFn = (ctx: XRPCReqContext) => ctx.req.ip
```

Comment:
Perhaps we drop a note nearby here re: `app.set('trust proxy', true)`?

Reply:
yeah good call 👍